src/jswrap_io.c

```diff
@@ -576,7 +572,11 @@ and there will be no debouncing.
 watch two pins with the same number - eg `A0` and `B0`.
 
  */
-JsVar *jswrap_interface_setWatch(JsVar *func, Pin pin, JsVar *repeatOrObject) {
+JsVar *jswrap_interface_setWatch(
+    JsVar *func,           //!< A callback function to be invoked when the pin state changes.
```

Comment:
You know these are documented in the JSON docs right above - and in more detail? This would be one of those times that having some JSON doc->doxygen formatter for Doxygen would be a huge help